composer.json

```diff
@@ -21,7 +21,13 @@
   "prefer-stable": true,
   "authors": [
     {"name": "Romans Malinovskis", "email": "romans@agiletoolkit.org", "homepage": "https://nearly.guru/" },
-    {"name": "Michael Voříšek", "homepage": "https://mvorisek.cz/"}
+    {"name": "Alain Belair"},
+    {"name": "Imants Horsts", "homepage": "https://darkside.lv/"},
+    {"name": "Francesco Danti", "homepage": "https://oracoltech.com/"},
+    {"name": "Michael Voříšek", "homepage": "https://mvorisek.cz/"},
+    {"name": "Philipp Grashoff"},
```

Comment:
@PhilippGrashoff Is it ok to add you as a 2nd major contributor and do you have a homepage to add?

Reply:
Current contributors here: https://github.com/atk4/infrastructure/blob/master/projects/agiletoolkit.org/github/agiletoolkit.org/github.tf#L6

But may not include our past contributors... 

https://github.com/atk4/ui/graphs/contributors

Reply:
Yes, I used the GH data in each repo and asked each contributor for an approval if they want to be added. All home pages collected except for "Philipp Grashoff". This is closed to me and merged already.